docs/home/api-info/webhooks.mdx

```diff
@@ -20,37 +20,39 @@ To enable this integration approach with Metriport:
 1. Set this endpoint URL on the [Developers page](https://dash.metriport.com/developers) in the developer dashboard,
    or via the [Update Settings endpoint](/home/api-reference/settings/post-settings);
 1. This will generate a webhook key that should be used to authenticate requests on
-   your app's endpoint (webhook).
+   your app's endpoint (webhook) - see [authentication](#authentication) and 
+   [generating a new webhook key](#generating-a-new-webhook-key) below.
 
 General requirements for the endpoint:
 
 1. Public endpoint accessible from the internet;
 1. Does not do an HTTP redirect (redirects will not be followed);
 1. Accepts a `POST` HTTP request;
-1. Verifies requests by comparing the HTTP header `x-webhook-key` with the webhook key;
+1. Verifies requests using the HTTP header `x-metriport-signature` - see [authentication](#authentication) below;
 1. Accepts and responds to a [`ping` message](#the-ping-message);
+1. Be [idempotent](https://en.wikipedia.org/wiki/Idempotence) - it should accept being called more
+   whan once with the same payload, with no changes to the end result.
```

Comment:
nit: whan -> than

Reply:
Is this indempotent requirement something that needs a todo item to communicate to cx's? Its a non trivial implementation detail on their end and they might not have known about that expectation before

Reply:
good catch, fixed!